tasks/translation-ollama/task.yaml

```diff
@@ -0,0 +1,19 @@
+id: translation-ollama
+name: LLM-translation template
+description: "This task is used for translating .pdf 
+  files into the desired language using the Ollama chatbot."
+is_active: true
+project_type: 
+param_groups:
+  - name: inputs
+    params:
+      - name: dataset
+        value: null
+        required: true
+        data_type: dataset
+        description: 'custom dataset with .pdf documents'
```

Comment:
There is no need to wrap strings in quotes in yaml file unless using special characters ":"